qt/widgets/common/src/FunctionTreeView.cpp

```diff
@@ -1988,6 +1990,9 @@ void FunctionTreeView::constraintChanged(QtProperty *prop) {
         emit parameterConstraintAdded(functionIndex, constraint);
```

Comment:
I'm wondering if we returned after this line we would avoid the crash. I haven't ever looked at this method, but after we've found the constraint property thats changed I wouldn't have thought we'd have to continue looping? I don't think you can change multiple constraints at a time. Maybe you'd have to test it out

Reply:
Ah right, I'll give it a try

Reply:
I can't find a situation where you can change multiple at a time, and I have checked that a return here does also stop a crash as the loop doesn't access memory it shouldn't. So going to change the code back and add a return instead for a cleaner solution. Thanks